### PR TITLE
Make code python3 compatible without running 2to3

### PR DESCRIPTION
**Pull request by CyberDem0n**
It was not producing correct code anyway...

### CODE REVIEW

File: test_syncobj.py [hunk 1]
```diff
@@ -336,7 +345,7 @@ def test_manyActionsLogCompaction():
 	newObjs = [o1, o2]
 	doTicks(newObjs, 10, stopFunc=lambda: o3._getLeader() is None)
 
-	for i in xrange(0, 500):
+	for i in range(0, 500):
```
Comment (bakwc) [inline test_syncobj.py]: may be xrange? seems like you defined it earlier for python3

**Comment by bakwc** [comment]
Thanks, great job! You are using range everywhere, may be it would be better to use xrange? There can be a performance issue when using range in python2.
Also coverage is crashing when running at travis python3.4 - I suggest to run test without coverage for python3.4.

**Comment by CyberDem0n** [comment]
> You are using range everywhere, may be it would be better to use xrange? There can be a performance issue when using range in python2.

It's not everywhere, but only in `benchmarks`. I don't think that you will see a big difference on small numbers. In all critical parts `xrange` is used (and iteritems for py2 instead of items).

> Also coverage is crashing when running at travis python3.4 - I suggest to run test without coverage for python3.4.

Sometimes it builds on travis are crashing without any visible reason and triggering build job solves it...
I did one more commit, if it wont help I will remove coverage for py34.

**Comment by CyberDem0n** [comment]
Funny, this time it crashed with python3.5

**Pull request merged**